engines/python/setup/djl_python/rolling_batch/rolling_batch.py

```diff
@@ -105,6 +106,79 @@ def get_content_type_from_output_formatter(output_formatter: Union[str,
     return None
 
 
+def _json_chat_output_formatter(token: Token, first_token: bool,
+                                last_token: bool, details: dict,
+                                generated_tokens: str):
+    """
+    json output formatter for chat completions API
+
+    :return: formatted output
+    """
+    choice1 = {
+        "index": 0,
+        "message": {
+            "role": "assistant",
+            "content": token.text
+        }
+    }
+    response1 = {
+        "id": f"chatcmpl-{uuid.uuid4()}",
+        "object": "chat.completion",
+        "choices": [choice1]  # Currently only support 1 choice
+    }
+    json_encoded_str = f"{json.dumps(response1, ensure_ascii=False)[:-6]}" if first_token else ""
+    json_encoded_str = f"{json_encoded_str}{json.dumps(token.text, ensure_ascii=False)[1:-1]}"
+    if last_token:
+        choice2 = {
+            "logprobs": {
+                "content": [{
+                    "logprob": token.log_prob,
+                    "token": token.text
+                }]
+            },
+            "finish_reason": details.get("finish_reason", None)
+        }
+        response2 = {
+            "choices": [choice2],
+            "usage": {
+                "completion_tokens": details["generated_tokens"]
```

Comment:
Can we also add `prompt_tokens` and `total_tokens`?

Reply:
Added